src/index.css

```diff
@@ -111,35 +111,49 @@ li {
 }
 
 .primary-button.btn,
-.primary-button {
-  background-color: var(--main-yellow);
+.primary-button,
+.secondary-button.btn,
+.secondary-button {
   padding: 12px 35px;
-  border-radius: 25px;
+  border-radius: 30px;
   color: var(--main-black);
   font-size: 24px;
   font-family: HelveticaNeueMed;
+  min-width: 200px;
+}
+
+.primary-button.btn,
+.primary-button {
+  background-color: var(--main-yellow);
   border-color: var(--main-yellow);
 }
 
+.secondary-button.btn,
+.secondary-button {
+  background-color: var(--main-blue);
+  border-color: var(--main-blue);
+}
+
 .primary-button.btn:hover,
 .primary-button:hover,
 .primary-button:not(:disabled):not(.disabled):active {
   background-color: var(--dark-yellow);
   border-color: var(--dark-yellow);
 }
 
-.secondary-button.btn,
 .secondary-button.btn:hover,
-.secondary-button,
 .secondary-button:hover,
 .secondary-button:not(:disabled):not(.disabled):active {
-  background-color: var(--main-blue);
-  padding: 12px 35px;
-  border-radius: 25px;
-  color: var(--main-black);
-  font-size: 24px;
-  font-family: HelveticaNeueMed;
-  border-color: var(--main-blue);
+  background-color: var(--blue);
+  border-color: var(--blue);
+}
+
+.primary-button:disabled .primary-button.btn:disabled,
```

Comment:
I'm not sure this selector will do anything? did you mean to have a comma in between